README.md

```diff
@@ -198,6 +204,43 @@ const player = new Player({
 });
 ```
 
+### Setting up a playback configurations
+
+If you need more option to playback configuration you can add new parameter `playbackConfig`
+
+#### Configuring through code
+
+```typescript
+// Simply pass the `playbackConfig` property to `PlaybackConfig` when instantiating a player.
+
+// With hooks
+import { usePlayer } from 'bitmovin-player-react-native';
+const player = usePlayer({
+  playbackConfig: {
+    // Optionally isAutoplayEnabled: Specifies whether autoplay is enabled. Default is false.
```

Comment:
Repeating the prop names in comments feels a little bit cumbersome here. You can just leave the description and default value.

```suggestion
    // Specifies whether the playback starts immediately after loading a source or not. Default is false.
```